src/screens/Projects.jsx

```diff
@@ -4,7 +4,7 @@ import Footer from "../components/footer";
 
 const Projects = () => {
   return (
-    <div className="wrapper">
+    <div className="wrapper projects-page">
```

Comment:
Should this be in the PR for about-me?

Reply:
No, but I made the change in the beginning because I thought I was going to work on my projects screen so I just saved the change. It's one less thing to do in the next step.